npm-shrinkwrap.json

```diff
@@ -6987,26 +6997,35 @@
       "integrity": "sha1-GZT/rs3+nEQe0r2sdFK3u0yeQaQ="
     },
     "husky": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/husky/-/husky-1.3.1.tgz",
-      "integrity": "sha512-86U6sVVVf4b5NYSZ0yvv88dRgBSSXXmHaiq5pP4KDj5JVzdwKgBjEtUPOm8hcoytezFwbU+7gotXNhpHdystlg==",
+      "version": "3.0.9",
```

Comment:
The previous PR added husky `3.0.9` to `package.json`, but didn't commit the corresponding changes to the `npm-shrinkwrap.json` file. This is harmless, but the changes should be committed.